ProcessMaker/processmaker
# FOUR-7379 | Review Template

# Description
Ticket: [FOUR-7379](https://processmaker.atlassian.net/browse/FOUR-7379)

As a Project Manager, I want to review the design and configuration of my template process so I can see exactly how the process is configured before creating a new process.

# How to Test
1. Go to branch `feature/FOUR-7379` in core.
2. Go to branch `feature/FOUR-7379` in Package Process Documenter. Install the package.
3. Go to Process Templates Listing
4. View Template Documentation
5. From the Template Documentation page, click on the template name in the breadcrumbs.
	- If a user has the Edit Process Templates permission, the template name breadcrumb should open the template in the Modeler.
	- If a user does not have the Edit Process Template permission, the template name breadcrumb should open a 'Not Authorized' page.

# Related Tickets & Packages
- [FOUR-7375](https://processmaker.atlassian.net/browse/FOUR-7375) | Process Templates
- [FOUR-7878](https://processmaker.atlassian.net/browse/FOUR-7878) | Unable to Access Template Documentation

# Code Review Checklist


[FOUR-7379]: https://processmaker.atlassian.net/browse/FOUR-7379?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-7375]: https://processmaker.atlassian.net/browse/FOUR-7375?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-7878]: https://processmaker.atlassian.net/browse/FOUR-7878?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
